runners/core-java/src/main/java/org/apache/beam/runners/core/GBKIntoKeyedWorkItems.java

```diff
@@ -49,7 +49,7 @@
         input.getWindowingStrategy().getWindowFn().windowCoder());
     PCollection<KeyedWorkItem<KeyT, InputT>> collection = PCollection.createPrimitiveOutputInternal(
```

Comment:
Another option is to use `PCollection.<KeyedWorkItem<...>>.createPrimitiveOutputInternal` and inline both variables, but this is ok too.